infrastructure/kube/keep-test/monitoring/README.md

```diff
@@ -0,0 +1,124 @@
+## Monitoring
+
+
+This documentation was made upon this [tutorial](https://devopscube.com/setup-prometheus-monitoring-on-kubernetes/)
+
+The Kubernetes Prometheus monitoring stack has the following components.
+
+1. Prometheus Server
+2. Alert Manager
+3. Grafana
+
+![Schema](https://devopscube.com/wp-content/uploads/2022/01/kubernetes-1024x498.png)
+
+### <u>_Prometheus_</u>
```

Comment:
Do we need `<u>` tags?
```suggestion
### Prometheus
```
or 

```suggestion
### _Prometheus_
```

Reply:
It's for underscore to split that README into two sections